### PR TITLE
Stats: Move Insights components to 2 column layout.

### PR DESCRIPTION
**Pull request by timmyc**
In order to use a masonry layout on the Stats pages, this branch converts most of the Insight page modules from "full width" to the 2-column concept seen on stat period pages:

__Before__
stats_ _trout_bummin_ _wordpress_com

__After__
stats_ _trout_bummin_ _wordpress_com

I took the styles of `<StatsTab />` for  `<480` and created an `is-compact` class to achieve this look.  I also removed "Today's Stats" - as it seems redundant to data that can be seen in depth on the 'Day' tab.

__To Test__
Open the Insights Page, view it at different viewport sizes.

### CODE REVIEW

File: client/my-sites/stats/stats-reach/index.jsx [hunk 1]
```diff
@@ -35,29 +35,29 @@ export const StatsReach = props => {
 		<div>
 			{ siteId && <QuerySiteStats siteId={ siteId } statType="statsFollowers" /> }
 			{ siteId && <QuerySiteStats siteId={ siteId } statType="statsPublicize" /> }
-			<SectionHeader label={ translate( 'Followers' ) } />
+			<SectionHeader label={ translate( 'Follower Totals' ) } />
```
Comment (a8ci18n) [inline client/my-sites/stats/stats-reach/index.jsx]: Hi! I've found a possible matching string that has already been translated **12** times:
`translate( ' Total Followers' )` *ES Score: 11.74*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/stats/insights-columns)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by folletto** [comment]
Very interesting change. I think it can work well, and takes away for sure some of the "too wide" effect there was before.

This also drops that style entirely right? So no difference anymore between mobile and desktop? We can later remove these CSS rules?

If we go in this direction, can we tweak the text to be more consistent? There are a LOT of sizes going on on this page.

I'd change:

```
.stats-tabs .stats-tab {
  font-size: 14px;
}

.stats-tabs .stats-tab.is-compact .label {
  text-transform: none;
  letter-spacing: 0;
  /* remove font-size */
}

.stats-tabs .stats-tab.is-compact .gridicon {
 margin: 2px 20px 0 20px; 
}
```

Which would look like this:

screen shot 2017-02-13 at 10 49 15

**Comment by timmyc** [comment]
> This also drops that style entirely right?

Unfortunately no - we still use the tabs in the chart, and on the stats overview page which lists all sites `/stats`.

> If we go in this direction, can we tweak the text to be more consistent?

Absolutely!  I was wondering that myself.  I'll update with the smaller text sizes.

**Comment by timmyc** [comment]
@folletto styles have been updated - curious if we should do something with the "Best Views Ever" text - perhaps we can put that in a pop-over, or do we need to change the styling at all?

stats_ _trout_bummin_ _wordpress_com

**Comment by folletto** [comment]
Ah, I missed that, thanks!

Let's align to the same title, use the new `$gray-text-min` color, and remove the `text-transform` and `letter-spacing`:

screen shot 2017-02-15 at 14 01 16

**Comment by timmyc** [comment]
Rebased to pull in `$gray-text-min` and updated - let me know if it looks good now @folletto

**Comment by folletto** [comment]
Design wise: 👍

**Pull request merged**